model/azure/registry.yaml

```
@@ -1,20 +1,91 @@
 groups:
-  - id: registry.azure.sdk
+  - id: registry.azure.client.sdk
     type: attribute_group
-    display_name: Azure SDK Attributes
+    display_name: Azure Client Library Attributes
     brief: >
-      This document defines generic attributes for Azure SDK.
+      This document defines generic attributes used by Azure Client Libraries.
     attributes:
       - id: az.service_request_id
         type: string
-        stability: experimental
+        stability: development
         brief: "The unique identifier of the service request. It's generated by the Azure service and returned with the response."
         examples:
           - "00000000-0000-0000-0000-000000000000"
       - id: az.namespace
         type: string
-        stability: experimental
+        stability: development
         brief: >
           [Azure Resource Provider Namespace](https://learn.microsoft.com/azure/azure-resource-manager/management/azure-services-resource-providers)
           as recognized by the client.
         examples: ["Microsoft.Storage", "Microsoft.KeyVault", "Microsoft.ServiceBus"]
+      - id: azure.client.id
+        type: string
+        stability: development
+        brief: The unique identifier of the client instance.
+        examples: ["3ba4827d-4422-483f-b59f-85b74211c11d", "storage-client-1"]
+  - id: registry.azure.cosmosdb
+    type: attribute_group
+    display_name: Azure Cosmos DB Attributes
+    stability: development
+    brief: >
+      This group defines attributes for Azure Cosmos DB.
+    attributes:
+      - id: azure.cosmosdb.connection.mode
+        type:
+          members:
+            - id: gateway
+              value: "gateway"
+              brief: Gateway (HTTP) connection.
+              stability: development
+            - id: direct
+              value: "direct"
+              brief: Direct connection.
+              stability: development
+        stability: development
+        brief: Cosmos client connection mode.
+      - id: azure.cosmosdb.request.charge
+        type: double
+        stability: development
+        brief: Request units consumed for the operation.
+        examples: [46.18, 1.0]
+      - id: azure.cosmosdb.request.body.size
+        type: int
+        stability: development
+        brief: Request payload size in bytes.
+      - id: azure.cosmosdb.request.regions_contacted
```

two thoughts:
* `contacted_regions` is more clearly "one thing" (when considering `_` vs `.`) compared to `regions_contacted`
  * maybe a tip we can give for using `_` is `adjective_noun`. since `adjective` can't be its own namespace.
  * similar to `returned_rows`, `speculative_execution`, `active_instance`
* ends in "s", which is kind of nice for array types(?)
* 

```suggestion
      - id: azure.cosmosdb.request.contacted_regions
```

---

awesome suggestion, created https://github.com/open-telemetry/semantic-conventions/issues/1755 so we don't forget to document it